tests/test_simple.py

```diff
@@ -182,7 +182,7 @@ def test_full_parse(self, module_name, count, expected_archive_link):
         html = importlib_resources.read_text(
             simple_data, f"archive_links.{module_name}.html"
         )
-        archive_links = simple.parse_archive_links(html)
+        archive_links, _ = simple.parse_archive_links(html)
```

Comment:
**Note**: This is the temporary, _Draft_, change I've put in place until some discussion is resolved. I will update these tests with the `index` return value alone in a subsequent PR.